src/com/trilead/ssh2/transport/KexManager.java

```diff
@@ -351,7 +394,14 @@ public static void checkKexAlgorithmList(String[] algos)
 
 			if ("diffie-hellman-group-exchange-sha256".equals(algo))
 				continue;
+			if ("ecdh-sha2-nistp256".equals(algo))
+                continue;
```

Comment:
please, can you format this piece of code?

Reply:
no formatter running when I run 'mvn clean test' ?

Reply:
no, a formatter is not running. I check the code the issue is that the file is tab formated and the new code is space formatted

Reply:
I am not a tab indent format but is the original file format so I have fixed the format